movie-app/src/header/NavBar.js

```diff
@@ -27,87 +27,90 @@ const NavBar = ({ setPage, movieSearch, setMovieSearch }) => {
     searchBar.current.focus();
     setActive(true);
     setFirstClick(true);
-  }
+  };
 
   const closeSwitch = (e) => {
     e.stopPropagation();
-    setActive(false)
-  }
+    setActive(false);
+  };
 
-  const handleSearchChange = (event)=>{
+  const handleSearchChange = (event) => {
     setSearchString(event.target.value);
-  }
-  const handleSubmit = ()=>{
-    titleChecker(searchString)
-  }
+  };
+  const handleSubmit = () => {
+    titleChecker(searchString);
+  };
 
-  const enterSubmit = (e)=>{
-    if(e.key==='Enter'){
-      titleChecker(searchString)
+  const enterSubmit = (e) => {
+    if (e.key === "Enter") {
+      titleChecker(searchString);
     }
-  }
+  };
 
   const applyStyles = (el) => {
     const styleApplier = () => {
       let currStyle = `btn-el_${el}`;
-      if(firstClick) {
-        if(active) { return currStyle + ` sidexside_${el}` } 
-        if(!active) { return currStyle +  ` stacked_${el}`}
+      if (firstClick) {
+        if (active) {
+          return currStyle + ` sidexside_${el}`;
+        }
+        if (!active) {
+          return currStyle + ` stacked_${el}`;
+        }
       }
       return currStyle;
-    }
+    };
     return styleApplier();
-  }
+  };
 
   return (
     <div className="header">
       <nav className="navigation">
         <div className="logo-container">
-          <img src={"logo.png"} alt="ToadTv logo" className="logo"  />
+          <a href="/">
+            <img src={"logo.png"} alt="ToadTv logo" className="logo" />
+          </a>
```

Comment:
Nice!  Users expect that clicking the site logo will return them to the homepage.